test/test_cohort_defaults.py

```diff
@@ -32,7 +32,44 @@
 
 FILE_FORMAT_1 = "patient_format1_%s.vcf"
 
-def test_default_filter_fn():
+def test_default_filter_fn_with_variant_count():
```

Comment:
Does this mostly repeat the `snv_count` one? Can they just be the same code called twice?